README.rst

```diff
@@ -274,6 +274,14 @@ Query records:
 
     sf.bulk.Account.query(query)
 
+Query all records:
```

Comment:
Please explain what this means (deleted records) in the README. This will be confusing.